tests/spread/integration/mawk/task.yaml

```diff
@@ -0,0 +1,8 @@
+summary: Integration tests for mawk
+
+execute: |
+  rootfs="$(install-slices mawk_bins)"
+
+  echo "test" | chroot "${rootfs}/" \
+  mawk '{print ($1 == "test" ? \
+  "PASS" : "FAIL")}'
```

Comment:
1. I would still keep the `chroot "${rootfs}/" mawk -W version` in addition to adding this more functional test here as well.
2. I would indent the second and third lines here, something like this perhaps?
```
  echo "test" | chroot "${rootfs}/" \
    mawk '{print ($1 == "test" ? \
    "PASS" : "FAIL")}'
```

Reply:
Also, is there anyway to coerce mawk to return a non-zero exit status if the test fails?
Otherwise, this test would continue to pass even if it triggered the `FAIL` print statement.